test/isochrone.cc

```diff
@@ -148,15 +148,15 @@ TEST(Isochrones, Basic) {
,52.056935],[5.137321,52.057547],[5.139321,52.057085],[5.139895,52.058363],[5.141624,52.059634],[5.141735,52.060937],[5.142808,52.061450],[5.144321,52.063187],[5.150021,52.057637],[5.151782,52.057398],[5.152321,52.056021],[5.152321,52.058197],[5.150894,52.058510],[5.148036,52.061937],[5.148321,52.062530],[5.149574,52.062684],[5.148835,52.063937],[5.148886,52.066372],[5.151091,52.068167],[5.152321,52.068296],[5.153321,52.067337],[5.154655,52.067271],[5.156321,52.065622],[5.158078,52.065937],[5.156855,52.066471],[5.155643,52.068259],[5.154321,52.068486],[5.152914,52.069937],[5.153321,52.070437],[5.157335,52.070937],[5.152321,52.071514],[5.151321,52.070638],[5.149107,52.072937],[5.151746,52.073512],[5.152321,52.075389],[5.153321,52.073864],[5.153943,52.075315],[5.155525,52.075733],[5.155380,52.077996],[5.153321,52.076444],[5.151876,52.077937],[5.153321,52.079469],[5.154548,52.079710],[5.155321,52.080761],[5.156130,52.080937],[5.156124,52.082134],[5.156835,52.082937],[5.156094,52.083164],[5.155657,52.082601],[5.154034,52.082224],[5.153669,52.081589],[5.152321,52.081506],[5.151583,52.080675],[5.149878,52.080380],[5.149526,52.079732],[5.148963,52.079937],[5.149164,52.081094],[5.151126,52.083132],[5.153768,52.083490],[5.153716,52.085937],[5.154647,52.087611],[5.155630,52.087937],[5.154321,52.089447],[5.151321,52.089430],[5.150858,52.089937],[5.151321,52.090504],[5.154088,52.090937],[5.153601,52.091217],[5.153321,52.093413],[5.152232,52.094026],[5.150321,52.093097],[5.149321,52.093808],[5.148321,52.093005],[5.147924,52.093540],[5.148321,52.094324],[5.149321,52.094187],[5.149858,52.095400],[5.151321,52.095321],[5.151894,52.097364],[5.154121,52.097937],[5.153491,52.098107],[5.153321,52.098841],[5.152974,52.098284],[5.151978,52.098280],[5.150908,52.098524],[5.150321,52.099513],[5.148321,52.099072],[5.146321,52.099355],[5.145730,52.099937],[5.146519,52.100937],[5.146321,52.101629],[5.145019,52.101239],[5.144912,52.099937],[5.144321,52.099476],[5.142828,52.100937],[5.143321,52.101667],[5.144485,52.101937],[5.142656,52.102937],[5.140321,52.102566],[5.140037,52.102221],[5.137321,52.102151],[5.136321,52.103072],[5.135894,52.102364],[5.134059,52.101937],[5.133781,52.099937],[5.134321,52.099377],[5.135597,52.099213],[5.135579,52.096937],[5.134662,52.096596],[5.134362,52.095896],[5.133518,52.095937],[5.132492,52.096108],[5.132153,52.096769],[5.130321,52.097087],[5.126772,52.099388],[5.124938,52.099320],[5.124251,52.098867],[5.124359,52.102937],[5.120321,52.103258],[5.119321,52.102698],[5.116321,52.105645],[5.115321,52.104355],[5.114321,52.104836],[5.112321,52.104914],[5.111321,52.104485],[5.110655,52.105271],[5.108321,52.106148]],"type":"LineString"},"type":"Feature"},{"geometry":{"coordinates":[[5.115328,52.078940]],"type":"MultiPoint"},"properties":{"location_index":0,"type":"snapped"},"type":"Feature"},{"geometry":{"coordinates":[5.115321,52.078937],"type":"Point"},"properties":{"location_index":0,"type":"input"},"type":"Feature"}],"type":"FeatureCollection"})";
```

Comment:
the same as above 
[merged.geojson.zip](https://github.com/valhalla/valhalla/files/6981559/merged.geojson.zip)